Index: deepin-lvm2/scripts/lvm2_monitoring_systemd_red_hat.service.in
===================================================================
--- deepin-lvm2.orig/scripts/lvm2_monitoring_systemd_red_hat.service.in
+++ deepin-lvm2/scripts/lvm2_monitoring_systemd_red_hat.service.in
@@ -1,6 +1,7 @@
 [Unit]
 Description=Monitoring of LVM2 mirrors, snapshots etc. using dmeventd or progress polling
 Documentation=man:dmeventd(8) man:lvcreate(8) man:lvchange(8) man:vgchange(8)
+ConditionPathExistsGlob=/sys/block/dm-*
 Requires=dm-event.socket
 After=dm-event.socket dm-event.service
 Before=local-fs-pre.target shutdown.target
